src/aks-preview/HISTORY.rst

```diff
@@ -13,6 +13,7 @@ Pending
 +++++++
 * Update --enable-advanced-network-observability description to note additional costs and add missing flag to create command.
 * Add etag support (--if-match, --if-none-match) to some aks commands for optimistic concurrency control.
+* Remove --enable-network-observability and --disable-network-observability from aks create and update commands.
```

Comment:
This is a breaking change as you removed several options.

Please set the version number as `5.0.0b1` (update `VERSION` in setup.py accordingly) and add `[BREAKING CHANGE]:` to the history note.